build.zig

```diff
@@ -41,6 +41,11 @@ pub fn build(b: *std.build.Builder) void {
     ) orelse .none;
     options.addOption(config.TracerBackend, "tracer_backend", tracer_backend);
 
+    const aof_override = b.option(bool, "aof", "Override AOF mode.");
+    const aof_recovery_override = b.option(bool, "aof_recovery", "Override AOF Recovery mode.");
+    options.addOption(?bool, "aof", aof_override);
+    options.addOption(?bool, "aof_recovery", aof_recovery_override);
```

Comment:
(It isn't set up set but eventually all the process configs will be set up this way.)

```suggestion
    const aof_override = b.option(bool, "config-aof", "Override AOF mode.");
    const aof_recovery_override = b.option(bool, "config-aof-recovery", "Override AOF Recovery mode.");
    options.addOption(?bool, "config_aof", aof_override);
    options.addOption(?bool, "config_aof_recovery", aof_recovery_override);
```